crates/revm/src/handler.rs

```diff
@@ -50,6 +50,12 @@ impl<'a, EXT, DB: Database> EvmHandler<'a, EXT, DB> {
                 } else {
                     Handler::mainnet_with_spec(cfg.spec_id)
                 }
```

Comment:
please add the bsc part comment of this `new` function

Reply:
Done